ubloxgps/gpsparser.go

```diff
@@ -0,0 +1,137 @@
+package ubloxgps
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+type GPSParser struct {
+	gpsDevice GPSDevice
+}
+
+// fix is a GPS location fix
+type Fix struct {
+	Valid     bool
+	Time      string
+	Latitude  string
+	Longitude string
```

Comment:
Would it be possible to convert these 3 strings to integers? That makes it a bit easier to work with them. Or maybe floats are more appropriate for lat/long, they seem to be used a lot to represent coordinates. Time could be represented as seconds since last midnight (so hours * 3600 + minutes * 60 + seconds).

Reply:
Why not represent the Time as a `Time` (https://golang.org/pkg/time/#Time), and the Latitude/Longitude as `float32`? That seems more like the expected types.